grails-app/views/program/_admin.gsp

```diff
@@ -70,41 +69,9 @@
             </div>
         </g:if>
 
-        <g:if test="${fc.userIsSiteAdmin()}">
+        <g:if test="${fc.userIsAlaOrFcAdmin()}">
         <div class="tab-pane" id="reporting">
-            <form>
```

Comment:
Just checking if this delete is intentional.

Reply:
Thanks Temi, yes, it's a hold over from the original RLP implementation where we modelled what is now a management unit as a sub-program.  This was to allow service providers to have different reporting dates to each other.
(Dave actually asked for the whole reporting section to be deleted in the original request but we (ALA admins) use the regenerate reports function when we update the program reporting config so moved it to ALA admin only)